addons/hr_holidays/models/hr_leave.py

```diff
@@ -523,7 +523,7 @@ def _compute_number_of_hours_text(self):
         for leave in self:
             leave.number_of_hours_text = '%s%s Hours%s' % (
```

Comment:
You could also have something like
```diff
@@ -521,9 +521,10 @@ class HolidaysRequest(models.Model):
     def _compute_number_of_hours_text(self):
         # YTI Note: All this because a readonly field takes all the width on edit mode...
         for leave in self:
-            leave.number_of_hours_text = '%s%s Hours%s' % (
+            leave.number_of_hours_text = '%s%g %s%s' % (
                 '' if leave.request_unit_half or leave.request_unit_hours else '(',
                 float_round(leave.number_of_hours_display, precision_digits=2),
+                _('Hours'),
                 '' if leave.request_unit_half or leave.request_unit_hours else ')')
```
to be more consistent with the module style (not really needed) and also fix the translation issue

Reply:
I fixed the translation in a separated commit since it has little to do with the bug in itself.